pkg/cloud/gen/main.go

```diff
@@ -886,17 +881,20 @@ func (g *{{.GCEWrapType}}) Insert(ctx context.Context, key *meta.Key, obj *{{.FQ
 		Service: "{{.Service}}",
 	}
 
-	klog.V(5).Infof("{{.GCEWrapType}}.Insert(%v, %v, ...): projectID = %v, ck = %+v", ctx, key, projectID, ck)
+	klog.V(5).Infof("{{.GCPWrapType}}.Insert(%v, %v, ...): projectID = %v, ck = %+v", ctx, key, projectID, ck)
```

Comment:
I think this is intentionally out of scope for this PR, but we may want to update the logs for NetworkServices API calls to be more reflective of the calls we're actually making.

Reply:
Corrected it to `Create` for TD resources.
Would you like to revert the change for `GCEWrapType` to `GCPWrapType`?

Reply:
Thanks! I think the `GCEWrapType` to `GCPWrapType` rename makes sense, good to leave that in place.